x/gov/types/params.go

```diff
@@ -105,12 +108,20 @@ func validateDepositParams(i interface{}) error {
 		return fmt.Errorf("invalid parameter type: %T", i)
 	}
 
-	if !v.MinDeposit.IsValid() {
-		return fmt.Errorf("invalid minimum deposit: %s", v.MinDeposit)
+	minDeposit := v.MinDeposit
+	if !minDeposit.IsValid() {
+		return fmt.Errorf("invalid minimum deposit: %s", minDeposit)
 	}
 	if v.MaxDepositPeriod <= 0 {
 		return fmt.Errorf("maximum deposit period must be positive: %d", v.MaxDepositPeriod)
 	}
+	minExpeditedDeposit := v.MinExpeditedDeposit
+	if !minExpeditedDeposit.IsValid() {
+		return fmt.Errorf("invalid minimum expedited deposit: %s", minExpeditedDeposit)
+	}
+	if minExpeditedDeposit.IsAllLTE(minDeposit) {
+		return fmt.Errorf("minimum expedited deposit %s must be greater than expedited deposit %s", minExpeditedDeposit, minDeposit)
+	}
```

Comment:
Does all proposals require to have `MinExpeditedDeposit`?

Reply:
Only expedited proposals end up using `MinExpeditedDeposit`. Other proposals use `MinDeposit`

Reply:
in that case, for the non expedited propsals, don't they end up going through validation here as well? ( i might be missing something)

Reply:
Ahh, this validation is for the params, not proposals.

Params are initialized globally and set by governance. Then, regular proposals use `MinDeposit` param and expedited proposals use `MinExpeditedDeposit` param

This is how the decision is made: https://github.com/osmosis-labs/cosmos-sdk/blob/6e9e75b2ab42414838ac98c23b860e634e982ea6/x/gov/types/proposal.go#L91

Reply:
Ahhh yeah this is for the params not the proposals itself!